packages/appstore/src/components/app.tsx

```diff
@@ -22,19 +23,21 @@ const App = ({ passthrough: { WS, root_store } }: TAppProps) => {
 
     return (
         <CashierStoreProvider store={root_store as any}>
-            <StoreProvider store={root_store as any}>
-                <main
-                    className={classNames('dashboard', {
-                        'theme--light': !ui.is_dark_mode_on,
-                        'theme--dark': ui.is_dark_mode_on,
-                        'dashboard-onboarding': window.location.pathname === routes.onboarding,
-                    })}
-                >
-                    <div className='dw-dashboard'>
-                        <Routes />
-                    </div>
-                </main>
-            </StoreProvider>
+            <CFDStoreProvider store={root_store as any}>
+                <StoreProvider store={root_store as any}>
```

Comment:
same for this

Reply:
didn't get your point @niloofar-deriv

Reply:
I just wanted to mention @hirad-deriv comment :laughing:

Reply:
```suggestion
                <StoreProvider store={root_store as TCoreStore}>
```